homeassistant/components/conversation/default_agent.py

```diff
@@ -133,6 +133,7 @@ async def async_process(
                     text,
                     context,
                     language,
+                    conversation_id,
```

Comment:
We should not pass conversation ID to the intent. Conversation ID should be used by the conversation agent to fill in slots based on sentence matching with history, something our `default_agent` does not support.